index.js

```diff
@@ -142,10 +143,18 @@ function buildRouteRate (pluginComponent, params, routeOptions) {
         }
 
         res.code(429)
-          .header('X-RateLimit-Limit', params.max)
+          .header('X-RateLimit-Limit', maximum)
           .header('X-RateLimit-Remaining', 0)
           .header('Retry-After', params.timeWindow)
```

Comment:
Use `x-rate-limit-reset` e.g https://developer.twitter.com/en/docs/basics/rate-limiting

Reply:
I have another pr coming soon for this header that isn't supported right now